activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -156,7 +156,7 @@ def write_entry(key, entry, options)
           expires_in = options[:expires_in].to_i
           if expires_in > 0 && !options[:raw]
             # Set the memcache expire a few minutes in the future to support race condition ttls on read
-            expires_in += 5.minutes
+            expires_in += 300
```

Comment:
Unless we pin this as a performance bottleneck, I would always go for the clearer code. IMO, 5.minutes is much clearer than 300. This is where I really don't like to manually apply .to_i. But I could live with an alias like `expires_in += 5.minutes.in_seconds`.